Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -26,6 +31,99 @@
 // 120; i.e., 120 units * 1/8 = 15 degrees." (http://doc.qt.io/qt-5/qwheelevent.html#angleDelta)
 constexpr double SCROLL_FRACTION_DEGREES = 15.;
 
+namespace
```

Comment:
That's a lot of code that is not bound to Qt. Maybe move this to UICommon?

Reply:
Yep, it makes sense to move it all out into something non-qt related. Going to move to Core/HW, as there's no UI-specific stuff in here either.

Reply:
Moving stuff has highlighted some problems, leaving other things (that assumed everything would be operating on mem1) in a pretty bad state - going to rework.

Reply:
Last sync should have addressed this.